### ISSUE TITLE
Add a link request

### ISSUE THREAD

**Issue by jcalfee** [ipfs/js-ipfs-bitswap#21]
Congratulations on having the top result at duckduckgo.com query: `bitswap protocol` .. Can you add a link to the main page to help users find more high level docs on the topic?  tq

**Comment by jbenet** [comment]
@jcalfee thanks-- we need a dedicated website and repo for the bitswap protocol. these do not exist yet.

right now (the sorry state of affairs is that) the best links are:
- https://ipfs.io/ipfs/QmR7GSQM93Cx5eAg6a6yRzNde1FQv7uL6X1o4k7zrJa3LX/ipfs.draft3.pdf
- https://github.com/ipfs/go-ipfs/tree/master/exchange/bitswap

@whyrusleeping we need to break bitswap out of go-ipfs.

**Comment by jbenet** [comment]
We need to:

- [ ] fill https://github.com/ipfs/specs/tree/master/bitswap (@jbenet @dignifiedquire @whyrusleeping)
- [ ] maybe create `ipfs/bitswap` repo to link to the spec and implementations in other langs. ?
- [ ] break go-bitswap out of go-ipfs (@whyrusleeping, @Kubuxu)

**Comment by diasdavid** [comment]
@dgrisham wanna work on this together? With the push of docs on https://github.com/ipfs/js-ipfs-bitswap/pull/76 and the recent refactor on https://github.com/ipfs/js-ipfs-bitswap/pull/142, I feel it is the right time to take advantage of the current modularity and how the code is organized to write the spec for Bitswap and each of the pieces.

The full docs/spec should go into https://github.com/ipfs/specs/tree/master/bitswap and I we could deliver the full spec with a talk so that it makes it easier to digest and also force us to think thoroughly how it gets the inner workings get explained and illustrated. Fun? :D

**Comment by dgrisham** [comment]
@diasdavid Yes! I appreciate the ping, I'd love to work together on this. I'll start reviewing the existing info you linked soon.

**Comment by dgrisham** [comment]
@diasdavid Hey! Want to get on a call this week and sync on this?

**Comment by diasdavid** [comment]
@dgrisham let's do this :) I'll be flying Tuesday, can we try Wed, Thu or Fri? Find my availability here: https://doodle.com/daviddias
